source/manual/alerts/nginx-5xx-rate-too-high-for-many-apps-boxes.html.md

```diff
@@ -11,6 +11,26 @@ review_in: 6 months
 You can view the 5xx logs across all machines on [this dashboard][nginx_5xx_grafana]
 (change the hostname to view different apps).
 
+## Spikes
+
+The alert should link to a graphite graph - often certain applications
+such as Whitehall can have spikes - if you can determine this is a spike
+it is best to acknowledge the alert and let a team that is working on the app
+know (or alert Platform Health).
+
+## Scaling up
+
+Sometimes a high 5xx rate can be because of a sudden increase in traffic to the
+site. You can use the [Nginx Requests][nginx-requests] dashboard to see if
```

Comment:
I wonder whether linking to `https://grafana.production.govuk.digital` as well would be a good idea?

Reply:
I couldn't get the dashboard to work in AWS (https://grafana.blue.production.govuk.digital/dashboard/file/nginx_requests.json?refresh=1m&orgId=1 "`Request Error`") so I thought I would leave it out (for now at least) and figure out why it was broken separately.

Reply:
If you specify a shorter time period, does it render for you? https://grafana.production.govuk.digital/dashboard/file/nginx_requests.json?refresh=1m&orgId=1&from=now-30m&to=now

Reply:
Oh, it does... hadn't tried that. I'll add a new PR which links to it.